### PR TITLE
XOR-MST

### PR DESCRIPTION
**Pull request by Mad-Penguin**
Testeado en
- [Club Assignment](https://codeforces.com/gym/103119/problem/C)
- [Xor-MST](https://codeforces.com/contest/888/problem/G)
- [XOR Partition](https://codeforces.com/contest/1849/problem/F)

### CODE REVIEW

**Comment by Mad-Penguin** [comment]
xd el dtype

File: content/graph/xorMST.h [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                if(trie[curr][b]) curr = trie[curr][b];
+                else curr = trie[curr][!b];
+            }
+            return trie[curr][2];
+        };
+
+        for(int i=L1; i<=R1; i++) add(a[i].fi, i);
+
+        int j;
+        int min_xor = -1;
```
Comment (DiegoEGM) [inline content/graph/xorMST.h]: dtype? xd

**Review by DiegoEGM — COMMENTED** [review]
(xd)^n

**Comment by DiegoEGM** [comment]
ta bien

File: content/graph/xorMST.h (hunk 1 above)
Comment (Mad-Penguin) [reply content/graph/xorMST.h]: 😮 True

**Comment by Mad-Penguin** [comment]
Olvide compilar, un seg

File: content/graph/xorMST.h [hunk 2]
```diff
@@ -0,0 +1,90 @@
+/**
+ * Author: Jose Soto
+ * Date: 2024-12-03
+ * License: ???
+ * Description: If problem requires long long make a custom case, is not tested for $LOG > 30$. 1-indexed (a and T). Given an array $a$ of $n$ integers computes a spanning tree of the complete graph where the weight between two different nodes $1 \leq i, j \leq n$ is $a_i \oplus a_j$. xor\_sum stores the weight of the MST. T is the tree.
```
Comment (Mad-Penguin) [inline content/graph/xorMST.h:5]: unu

**Comment by DiegoEGM** [comment]
ta bien x2

**Pull request merged**